### PR TITLE
Update Default Configs

### PR DESCRIPTION
**Pull request by StudentAlleg**
See also https://github.com/Mykindos/BetterPvP-Private/pull/37

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested my changes.

### CODE REVIEW

File: clans/src/main/resources/configs/config.yml [hunk 1]
```diff
@@ -160,28 +169,28 @@ command:
...
-      requiredRank: ADMIN
+      requiredRank: PLAYER
       withdraw:
         enabled: true
-        requiredRank: ADMIN
+        requiredRank: PLAYER
       deposit:
         enabled: true
-        requiredRank: ADMIN
+        requiredRank: PLAYER
```
Comment (Mykindos) [inline clans/src/main/resources/configs/config.yml:178]: currently still admin only, only purpose for the bank is to store gold to collect from sold auctions

File: clans/src/main/resources/configs/config.yml [hunk 2]
```diff
@@ -160,28 +169,28 @@ command:
...
     withdraw:
       enabled: true
-      requiredRank: ADMIN
+      requiredRank: PLAYER
     deposit:
       enabled: true
-      requiredRank: ADMIN
+      requiredRank: PLAYER
```
Comment (Mykindos) [inline clans/src/main/resources/configs/config.yml:184]: same

File: clans/src/main/resources/configs/config.yml [hunk 3]
```diff
@@ -208,13 +217,31 @@ command:
...
     tutorial:
       enabled: true
-      requiredRank: ADMIN
+      requiredRank: PLAYER
     vault:
+      enabled: true
+      requiredRank: PLAYER
```
Comment (Mykindos) [inline clans/src/main/resources/configs/config.yml:226]: admin, gets accessed via clan core

**Review by Mykindos — CHANGES_REQUESTED** [review]

File: clans/src/main/resources/configs/config.yml (hunk 3 above)
Comment (StudentAlleg) [reply clans/src/main/resources/configs/config.yml]: gotcha

**Pull request merged**